llvm/test/CodeGen/Generic/zero-sized-array.ll

```diff
@@ -1,4 +1,5 @@
 ; RUN: llc < %s
+; RUN: llc -O0 -mtriple=aarch64 -global-isel < %s
 ; PR9900
 
```

Comment:
There is not really such a thing as a generic codegen test. You have to have a target, and this demonstrates yet another example of why. This will fail if aarch64 isn't built. You should add a copy in test/AArch64/GlobalISel and not add a target run line here

Reply:
Okay. Moved and minimized it into `test/AArch64/GlobalISel` folder.